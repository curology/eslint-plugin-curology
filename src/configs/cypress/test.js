const { cypress } = require(".");
const { generateTest } = require("../tests/utils");

describe("eslint-plugin-curology", () => {
  describe("cypress config", () => {
    generateTest(__dirname, cypress);
  });
});
