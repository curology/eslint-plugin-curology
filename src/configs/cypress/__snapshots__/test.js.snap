// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`eslint-plugin-curology cypress config serializes to a snapshot 1`] = `
Object {
  "env": Object {
    "cypress/globals": true,
  },
  "globals": Object {},
  "ignorePatterns": Array [],
  "parserOptions": Object {},
  "plugins": Array [
    "cypress",
  ],
  "rules": Object {
    "@typescript-eslint/ban-ts-ignore": Array [
      0,
    ],
    "@typescript-eslint/camelcase": Array [
      0,
    ],
    "@typescript-eslint/explicit-function-return-type": Array [
      0,
    ],
    "@typescript-eslint/no-unused-vars": Array [
      1,
      Object {
        "argsIgnorePattern": "^_",
      },
    ],
    "camelcase": Array [
      0,
    ],
    "cypress/assertion-before-screenshot": Array [
      2,
    ],
    "cypress/no-assigning-return-values": Array [
      1,
    ],
    "cypress/no-async-tests": Array [
      2,
    ],
    "cypress/no-force": Array [
      1,
    ],
    "cypress/no-pause": Array [
      2,
    ],
    "cypress/no-unnecessary-waiting": Array [
      1,
    ],
    "cypress/require-data-selectors": Array [
      0,
    ],
    "import/order": Array [
      2,
      Object {
        "groups": Array [
          Array [
            "builtin",
            "external",
          ],
          "internal",
          Array [
            "parent",
            "sibling",
          ],
        ],
        "newlines-between": "always",
        "pathGroups": Array [
          Object {
            "group": "parent",
            "pattern": "{constants,integration,fixtures}/**",
            "position": "before",
          },
        ],
        "pathGroupsExcludedImportTypes": Array [],
        "warnOnUnassignedImports": false,
      },
    ],
    "no-console": Array [
      "off",
    ],
  },
  "settings": Object {},
}
`;
